src/crypto/crypto.cc

```diff
@@ -65,46 +64,38 @@ uint64_t Crypto::unique( void )
 {
   static uint64_t counter = 0;
   uint64_t rv = counter++;
-  if ( counter == 0 ) {
-    throw CryptoException( "Counter wrapped", true );
-  }
+  if ( counter == 0 ) { throw CryptoException( "Counter wrapped", true ); }
```

Comment:
Can clang-format force a newline after open curly brace, so this stays on multiple lines?

Reply:
Done.